docs-content/getting-started/3_client-sdk/7_replay-configuration/opentelemetry.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Collecting OpenTelemetry Data in Client-Side Applications
+slug: salesforce-lwc
+createdAt: 2024-08-23T22:55:19.000Z
+updatedAt: 2024-08-23T22:55:19.000Z
+---
+
+## Collecting OpenTelemetry Data in Client-Side Applications
+
+Highlight.io's JavaScript SDK provides built-in support for collecting OpenTelemetry data from client-side applications. This feature allows you to seamlessly integrate OpenTelemetry tracing into your web applications, providing valuable insights into application performance and user interactions.
+
+### Enabling OpenTelemetry Tracing
+
+To enable OpenTelemetry tracing in your application using the Highlight.io SDK, you need to set the `enableOtelTracing` configuration option when initializing the SDK. The OpenTelemetry browser instrumentation is currently in beta, so you need to explicitly enable it when calling [`H.init`](/docs/sdk/client#Hinit).
+
+```ts
+H.init({
+  // ...
+  enableOtelTracing: true
+})
+```
+
+## Types of OpenTelemetry Data Collected
+
+When OpenTelemetry tracing is enabled, the Highlight.io SDK automatically collects various types of data using built-in instrumentations. These include:
+
+1. Document Load: Captures timing information about the document load process, including navigation start, DOM content loaded, and load event end times.
+
+2. User Interactions: Records spans for user interactions such as clicks, including details about the target element and the type of interaction.
+
+3. XMLHttpRequest (XHR): Traces outgoing XHR requests, including the URL, method, and timing information.
+
+4. Fetch Requests: Similar to XHR, this captures information about outgoing fetch requests, including URL, method, and timing data.
+
+5. GraphQL Operations: For applications using GraphQL, the SDK can capture and parse GraphQL operations, providing insights into query and mutation performance.
+
+These auto-instrumentations provide a comprehensive view of your application's performance from the client-side perspective, allowing you to identify bottlenecks, track user interactions, and monitor network requests without manual instrumentation.
+
+## Connecting Client and Server Traces
+
+One of the powerful features of OpenTelemetry is the ability to connect traces across different services and environments, including from the client to the server. Highlight.io's SDK facilitates this connection through context propagation. This allows you to create end-to-end traces that span from user interactions in the browser all the way to your backend services.
```

Comment:
is there somewhere we should link to this?

Reply:
Are you thinking we should link out to OTeL docs on context propogation?